hir/src/asm/stack.rs

```diff
@@ -458,6 +458,19 @@ impl<'a, E: StackElement, T: ?Sized + Stack<Element = E>> DebugStack<'a, T> {
         self.limit = Some(n);
         self
     }
+
+    pub fn pretty(&self) -> String {
```

Comment:
I think we should make this a `Display` impl, and base it on the existing `Debug` impl. The only difference between this and `Debug` seems to be how we format each stack element, which is certainly useful, but I'd rather use `Display` for it, since it is far more ergonomic than building strings directly, and you get `to_string` for free.

Reply:
Good point! Done in https://github.com/0xPolygonMiden/compiler/pull/237/commits/8ca444b1a4e62a4faf11005216554dc58cac6a1a